<launch>
  
  <include file="$(find robot_utils)/launch/joystick.launch"/>
  
  <include file="$(find robot_utils)/launch/frames.launch"/>
  
  <node pkg="ur_lightweight_driver" type="ur_rtde_driver.py" name="ur_rtde_driver" output="screen">
      <param name="config_file" value="$(find ur_lightweight_driver)/src/control_loop_configuration.xml"/>
  </node>
  
  <node pkg="ur_lightweight_driver" type="command_manager.py" name="command_manager" output="screen" />
  
  <node pkg="robot_utils" type="joystick_commander.py" name="joystick_commander" output="screen" />

  <node pkg="robot_utils" type="block_commander.py" name="block_commander" output="screen" />

  <include file="$(find robot_utils)/launch/cameras.launch"/>

</launch>